app/src/main/res/layout-land/topic_info_fragment.xml

```diff
@@ -35,7 +35,7 @@
         android:fontFamily="sans-serif"
         android:text="@{viewModel.topic.name}"
         android:textColor="@color/oppia_primary_text"
-        android:textSize="24sp"
+        android:textSize="23sp"
```

Comment:
This change is incorrect, we should fix this without changing the text size otherwise the UI won't be following what mocks/designs suggest us.

Also, this 23sp will work only for big devices, what if the device is small then again this will be a problem

Undo all this change try to fix it by applying some other approach.

Reply:
Okay, I'll solve it by something else.